src/System/UnixSystem.php

```diff
@@ -33,6 +33,24 @@ public function getCurrentUserHome()
     }
 
     /**
+     * @param string $cmdName
+     * @return boolean
+     */
+    public function checkIfCommandExists($cmdName)
+    {
+        if (!function_exists('exec')) {
+            return false;
+        }
+
+        $sysResponse = exec(
+            'PATH='.getenv('PATH').'; '.
+            "if command -v ".$cmdName." >/dev/null 2>&1; then echo \"true\"; else echo \"false\"; fi;"
+        );
+
+        return filter_var($sysResponse, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE);
```

Comment:
The command should not return null, but the flags passed to `filter_var` allows returning null.